packages/webapp/src/routes/index.jsx

```diff
@@ -509,7 +509,13 @@ const Routes = () => {
             />
             <Route path="/tasks/:task_id/harvest_uses" exact component={HarvestUses} />
             <Route path="/tasks/:task_id/abandon" exact component={TaskAbandon} />
-            <Route path="/map" exact component={Map} />
+            <Route
+              path="/map"
+              exact
+              component={(history) => (
```

Comment:
@antsgar I think history just needs to be destructured here is all to fix the errors. My only real change request

Reply:
Ahhh good catch! Will fix. Re passing isCompactSideMenu -- if the styling is done properly, we shouldn't even need to be aware of the width of the side menu. Sadly the absolute positioning of elements here in the map makes it hard to center stuff without relying on that number. Even then, instead of just drilling that down we should store it globally with React Context, but I figured this was okay for now.

Reply:
@Duncan-Brain should be fixed now, thank you for catching it!